.github/workflows/tagcheck.yml

```
@@ -0,0 +1,20 @@
+name: Linktagcheck
+on:
+  pull_request:
+    types: [opened, synchronize]
+permissions:
+  contents: read
+jobs:
+  Linktagcheck:
```

Is it okay if we use pascal case for the build name so it's `LinkTagCheck`?

```suggestion
  LinkTagCheck:
```